thavalon-webapp/src/components/Home.tsx

```diff
@@ -1,7 +1,28 @@
 import React from 'react';
+import { AccountManager, HttpResponse } from '../utils/AccountManager';
+import { GameSocket } from '../utils/GameSocket';
 
 export function Home(): JSX.Element {
+    const accountManager = AccountManager.getInstance();
+
+    function makeGame() {
```

Comment:
All of this code can be removed with PR #55

Reply:
Looks like this code is still here. If you need it for testing we can leave it in this PR.

Reply:
yeah i deleted then re-added to test and forgot to delete again... will remove